1/SumOfEvenNumbers.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgileFreaks_test1
+{
+    public class SumOfEvenNumbers
+    {
+        private int s;
+
+        public SumOfEvenNumbers()
+        {
+            s = 0;
+        }
+
+        public int computeEvenNumbersSum(int[] array)
+        {
+            for (int i = 0; i < array.Length; i++)
+                if (isEven(array[i]))
+                    addToSum(array[i]);
+            return s;
```

Comment:
You are using an instance variable that gets modified when this method is executed. If you call this method more than once, you will get invalid results.